docs/deploying-airbyte/on-oci-vm.md

```diff
@@ -20,14 +20,52 @@ Select the Subnet &gt; Security List &gt; Add Ingress Rules
 
 ![](../.gitbook/assets/OCIScreen3.png)
 
-## Login to the Instance/VM with the SSH key and 'opc' user
+### Connection Method 1 : Create SSH Tunnel via a Bastion Host to Login to the Instance 
+
+it is highly recommended to not have a Public IP for the Instance where you are running Airbyte\).
+
+#### SSH Local Port Forward to Airbyte VM
+
+From your local workstation
+
+```text
+ssh opc@bastion-host-public-ip -i <private-key-file.key> -L 2200:oci-private-instance-ip:22
+ssh opc@localhost -i <private-key-file.key> -p 2200
+```
+
+### Connection Method 2 : Create OCI Bastion Host Service to Login to the Instance 
+
+![OCIScreen13](https://user-images.githubusercontent.com/39692236/142787487-71d464de-38ec-4198-944a-328ccc1a0906.png)
+
+
+#### Create Bastion Host Service from OCI Console
+<img width="1257" alt="OCIScreen5" src="https://user-images.githubusercontent.com/39692236/142786778-99925599-1b68-444a-be76-2a57bd74b5e2.png">
```

Comment:
I'm not sure that this form of image formatting will work. Can you commit the image files under `docs/gitbook/assets` and directly reference them?